example/tck_services.py

```diff
@@ -0,0 +1,115 @@
+"""
+TJS 2020 apache 2.0
+"""
+
+import sys
+from logging import getLogger
+
+import grpc
+import pytest
+
+from cloudstate.cloudstate import CloudState
+from functiondemo.function_definition import definition, definition2
+from functiondemo.functiondemo2_pb2 import FunctionRequest2
+from functiondemo.functiondemo2_pb2_grpc import FunctionDemo2Stub
+from functiondemo.functiondemo_pb2 import FunctionRequest, AddToSum, FunctionResponse
+from functiondemo.functiondemo_pb2_grpc import FunctionDemoStub
+from shoppingcart import shopping_cart_entity
+from shoppingcart.shoppingcart_pb2 import AddLineItem, GetShoppingCart
+from shoppingcart.shoppingcart_pb2_grpc import ShoppingCartStub
+
+logger = getLogger()
+
+if __name__ == "__main__":
```

Comment:
Since we don't have a reference TCK for StatelessFunctions yet implemented, wouldn't it be more interesting to turn this into unit testing / integration? Wdyf?

Reply:
work in progress moving to unit tests.

Reply:
looks like the proxy needs to be running to get any meaningful test of the functionality. I think the only solution would be to pull/boot the docker container during ci, which might be annoyingly slow.

Reply:
I think it's OK to have the CI pulling docker images, hence the proxy, to test functionality; we do it at different other points. If not a language support library is doing it, the main project does it when it runs the TCK for all known support libraries; which might be too late depending on the changes made.

There is also: https://observablehq.com/@marcellanz/cloudstate-go-tck-verification that runs the TCK nightly.